e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs

```diff
@@ -35,6 +35,11 @@ public class PublishSingleNodesTestTheory
         public async Task Test_CollectOAuthToken() {
             var token = await TestHelper.GetTokenAsync(_context);
             Assert.NotEmpty(token);
+
+            // We will wait for microservices of IIoT platform to be healthy.
+            var millisecondsDelay = 600 * 1000;
```

Comment:
Please use constant TestConstants.MaxDelayToEdgeModulesToBeLoadedInMilliseconds instead, and rename the constant to `MaxDelayDeploymentToBeLoadedInMilliseconds`

Reply:
Done.